Sources/SwifterAuth.swift

```diff
@@ -73,15 +73,12 @@ public extension Swifter {
      Begin Authorization with a Callback URL
      
      - Parameter presentFromViewController: The viewController used to present the SFSafariViewController.
-     The UIViewController must inherit SFSafariViewControllerDelegate
-     
      */
     
     #if os(iOS)
     func authorize(withCallback callbackURL: URL,
                    presentingFrom presenting: UIViewController?,
                    forceLogin: Bool = false,
-                   safariDelegate: SFSafariViewControllerDelegate? = nil,
```

Comment:
This is a breaking change. We can't just rip this out since there are clients using this, unless we move this change to be a major update change. I suggest keeping the original code, but fallback to a default delegate when the user doesn't provide any safari delegate.